src/native_main.nim

```diff
@@ -73,6 +75,34 @@ proc findUserConfigFile(): Option[string] =
 
     return config_path
 
+proc removeFileAttribute(file, attr: string): int =
+    var xattrErr = 0
+    var xattrCmd = ""
+
+    if DEBUG:
+        xattrCmd = quoteShellCommand(["xattr", "-d", attr, file])
```

Comment:
Does xattr require root permissions?

What happens if you try to remove attributes that don't exist on the file?